packages/class-validator/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@hono/class-validator",
+  "packageManager": "yarn@4.0.2",
+  "description": "Validator middleware using Zod",
+  "version": "0.1.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "scripts": {
+    "test": "vitest --run",
+    "build": "rimraf dist && tsc",
```

Comment:
Can you use `tsup` to build? You can refer to this project: https://github.com/honojs/middleware/tree/main/packages/hello